cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -45,8 +45,6 @@ private[akka] object BootstrapCoordinator {
 
   object Protocol {
     final case class InitiateBootstrapping(selfContactPoint: Uri)
-    sealed trait BootstrappingCompleted
-    final case object BootstrappingCompleted extends BootstrappingCompleted
```

Comment:
I don't quite get how adding that all contact points setting makes this response superflous.

Reply:
It was an unrelated change. This always went to dead letters